frontend/src/styles.css

```
@@ -3006,27 +3258,213 @@ ul {
+.folders-rail__toggle {
+  width: 44px;
+  flex: 0 0 44px;
+  border-radius: 12px;
+  margin: 2px 6px 2px 2px;
+  align-self: flex-start;
+}
+
+.folders-rail__toggle-icon {
+  width: 16px;
+  height: 16px;
+  fill: currentColor;
```

Stylelint (17.6.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)